vite.config.ts

```diff
@@ -1,14 +1,26 @@
 import { resolve } from "path"
 import { defineConfig, ViteUserConfig } from "vitest/config"
 
+const entryPointToFolder = {
```

Comment:
should we instead just have a list of entry points and do the mapping programatically where needed?